src/index.html

```diff
@@ -4,9 +4,188 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap&subset=cyrillic" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.css">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="header__logo-link">
+        <img src="images/logo.png" alt="Moyo logo">
+      </a>
+      <nav class="navigation">
+        <ul class="navigation__list" >
+          <li class="navigation__element"><a class="navigation__reference-group is-active" href="">APPLE</a></li>
```

Comment:
1. переносы длинных строк
2. чем проще тем лучше
```suggestion
          <li class="nav__item">
            <a class="nav__link nav__link--is-active" href="">APPLE</a>
          </li>
```